ostree-ext/src/container/unencapsulate.rs

```diff
@@ -191,8 +191,59 @@ pub async fn unencapsulate(repo: &ostree::Repo, imgref: &OstreeImageReference) -
     importer.unencapsulate().await
 }
 
+trait Decompressable: Read + Send + 'static {
+    fn get_inner_mut(&mut self) -> &mut (dyn Read);
```

Comment:
The concrete versions (e.g. [this one](https://docs.rs/zstd/latest/zstd/stream/read/struct.Decoder.html#method.get_mut)) spell it `&mut R` which seems clearer to me.

Reply:
Ahh right of course. Yep it needs to be as is then.